packages/marshal/src/helpers/remotable.js

```diff
@@ -202,7 +203,7 @@ export const RemotableHelper = harden({
     if (
       !(
         check(
-          typeof candidate === 'object' || typeof candidate === 'function',
+          isObject(candidate),
```

Comment:
This check is not equivalent for `null`, making the check 2 lines down redundant

Reply:
Good catch!

Reply:
Done